### ISSUE TITLE
BUAP ANC import missing item

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2982]
We need to revisit the ones @liviah848 didn't import, too.

I have one now that is missing that maybe we can explore first: ANC 30876. 
@urapadmin I find this in the ANC database no problem and it is listed as OP 145 Locus 20 Lot 28.
Other things from that locus and lot WERE imported properly. Please figure it out!

**Comment by urapadmin** [comment]
one of over 440 VOID material records which the code skips.

**Comment by urapadmin** [comment]
btw. 
There are still a few warnings:
ERROR:   Line 34169, Site ANC, ANC# 38567: Duplicate ANC#. Line skipped.
WARNING: Line 32883, ANC# 38043: LOCUS 15b is not numeric.
WARNING: Line 19390, ANC# 21636: material is empty.
WARNING: Line 40460, ANC# 31427: material is empty.

**Comment by urapadmin** [comment]
Kiosk Shared Files\field projects\BUAP\ANC import\ANC database_3-23-2012 for Kiosk Import.fmp12
on the Google Drive is the database we use for imports.

**Comment by urapadmin** [comment]
NOT A BUG!

**Comment by lbestock** [comment]
Not a bug, just a pain in the tuchas! What the heck! VOID in this case is definitely a mistaken way of indicating "composite, has no single material."

But those ones with material empty also seem not to have come in? At least a Q&V search on them does not find them for me.

**Comment by urapadmin** [comment]
should they have been added as collected materials? At least they would not have failed to come in at the stage where I skip voids. I can find them in the internal Kiosk copy of the anc database in buap.

**Comment by lbestock** [comment]
Why did it try to bring in those ones yet? At least "WARNING: Line 40460, ANC# 31427: material is empty" is from a locus that hasn't been created yet.

**Comment by lbestock** [comment]
From @liviah848 email:

I am running into a few issues here with the ANC records. For locus 10, it seems that I have input this into KIOSK, and I cannot add lot 8 to this locus because there is already a lot 8 for locus 9. For many of the other ANC# which are missing, I have checked and both the locus and lot they are from are inputted into KIOSK:

- [x] 43622, locus 10 lot 8, ceramic, 19.XII.11 ---- lot 8 already assigned to locus 9 in the locus log
- [x] 43630, locus 7 lot 9, ceramic, 19.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43631, locus7 lot 9, human bone, 19.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43648, locus 7 lot 9, faunal bone, 19.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43652, locus 10 lot 14, ceramic, 20.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43673, locus 10 lot 14, human bone, 20.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43674, locus 10 lot 14, painted bone, 20.XII.11 -- this locus and lot already exist in KIOSK
- [x] 43894, locus 14 lot 28, ceramic, 28.XII.11 -- lot 28 already assigned to locus 8 in locus log
- [x] 43895, locus 15 lot 29, ceramic, 28.XII.11 -- lot 29 already assigned to locus 31 in locus log 
- [x] 43896, locus 14 lot 28, human bone, 28.XII.11 -- lot 28 already assigned to locus 8 in locus log
- [x] 44003, locus 15 lot 29, human bone, 28.XII.11 -- lot 29 already assigned to locus 31 in locus log 
- [x] 45750, locus 29 lot 32, human bone, 7.I.12 -- this locus and lot already exist in KIOSK
- [x] 46056, locus 29 lot 35, human bone, 7.I.12 -- lot 35 already assigned to locus 36 in locus log 
- [x] 46082, locus 31 lot 34, ceramic vessel, 8.I.12 -- this locus and lot already exist in KIOSK
- [x] 46085, locus 36 lot 35, ceramic vessel, 8.I.12 -- this locus and lot already exist in KIOSK

**Comment by urapadmin** [comment]
you are talking to me from the other room

**Comment by lbestock** [comment]
I am talking to myself! Sort of.
One can make a duplicate lot number, so I have just created 179-10-8. Now we'll see if that leads to 43622 being brought in. But that's not the most common problem here, still other things to check.

**Comment by lbestock** [comment]
The duplicate lot creation did bring in 43622. 
I made these things checkboxes so I can mark them off. Alas I waited so long on this that there's no way Livia is going to remember precise things. Hmpf.
I'm now checking the others, and I see that the next three ARE there when I check in Q&V. And then there are two CM in 179-7 that are hand-created in FileMaker and have hierarchical numbers rather than ANC#s. @liviah848 can you clarify?

**Comment by lbestock** [comment]
All of these are in there now. Most required creation of a duplicate lot. Those that were initially missing I still don't understand - the ones with the locus and lot present were all already there when I started to go through this. But I'm guessing this is a user workflow thing (looking for them before synchronization perhaps?) and not a Kiosk thing. I've checked with @liviah848 to make sure she sets GitHub to alert her when she's mentioned! But this can be closed.
